src/vs/workbench/contrib/notebook/browser/view/cellParts/markupCell.ts

```diff
@@ -262,8 +265,9 @@ export class MarkupCell extends Disposable {
 		const element = DOM.$('div');
 		element.classList.add('cell-collapse-preview');
 		const richEditorText = this.getRichText(this.viewCell.textBuffer, this.viewCell.language);
-		domSanitize.safeSetInnerHtml(element, richEditorText);
+		domSanitize.safeSetInnerHtml(element, richEditorText, getTokenizedPreviewSanitizerConfig());
```

Comment:
Worth checking if you need the sanitize at all in this flow. If the html string just the output of `tokenizeLineToHTML`, I *think* it should be safe but Alex D or Martin should be able to confirm

Reply:
My thought is that so long as we are only doing syntax highlighting, this would be the same as rendering the content in a normal text editor, which we already do for untrusted content. However good to double check